scripts/build/termux_step_massage.sh

```diff
@@ -80,6 +80,12 @@ termux_step_massage() {
 		else
 			echo "TERMUX_SUBPKG_DEPENDS=\"$TERMUX_PKG_NAME\"" >> "$_DEVEL_SUBPACKAGE_FILE"
 		fi
+		if [ -n "$TERMUX_PKG_DEVPACKAGE_BREAKS" ]; then
```

Comment:
These new variables need to be initialized in `termux_step_setup_variables.sh`. Otherwise the build of any package other than `zlib` will cause.
```
scripts/build/termux_step_massage.sh: line 83: TERMUX_PKG_DEVPACKAGE_BREAKS: unbound variable
```